docs/advanced/settings.md

```diff
@@ -445,7 +445,7 @@ Default: `10`
 
 _Available as of v1.3.0_
 
-This setting allows you to define the number of minutes Harvester waits before deleting a support bundle that has been packaged but not downloaded. The minimum value is `30`.
+This setting allows you to define the number of minutes Harvester waits before deleting a support bundle that has been packaged but not downloaded, failed to download or retained. The minimum value is `30`.
```

Comment:
```suggestion
This setting allows you to define the number of minutes Harvester waits before deleting a support bundle that has been packaged but not downloaded (either deliberately or unsuccessfully). The minimum value is `30`.
```

Reply:
I'm not sure if we sure mention the "retained" part because it might confuse users. If something is marked for retention, why should the system automatically delete it?